src/appkit.cs

```diff
@@ -23671,4 +23676,10 @@ interface NSOpenGLLayer
 		void Draw (NSOpenGLContext context, NSOpenGLPixelFormat pixelFormat, double t, ref CVTimeStamp ts);
 	}
 
+	[Protocol (IsInformal=true)]
```

Comment:
@spouliot Filed - https://bugzilla.xamarin.com/show_bug.cgi?id=44904

How should I proceed on this? [Protocol (Informal=True)] or a category (since we're not getting auto complete anyway)?